AEM/mlperf/README.md

```diff
@@ -7,9 +7,720 @@ translation, and others. The benchmark is representative of real-world
 workloads and as a fair and useful way to compare the performance of different
 machine learning systems.
 
+In this document, we'll show how to run Intel MLPerf v3.1 submission with Intel
+optimized Docker images.
+
+## Intel Docker Images for MLPerf
+
+The Intel optimized Docker images for MLPerf v3.1 can be built using the Dockerfiles.
+Example for building docker image with Dockerfile:
+```
+cd <THIS_REPO>/closed/Intel/code/resnet50/pytorch-cpu/docker/
+
+bash build_resnet50_contanier.sh
+```
+
+## HW configuration:
+
+| System Info     | Configuration detail                 |
+| --------------- | ------------------------------------ |
+| CPU             | SPR                       |
+| OS              | CentOS  Stream 8                     |
+| Kernel          | 6.1.11-1.el8.elrepo.x86_64 |
+| Memory          | 1024GB (16x64GB 4800MT/s [4800MT/s]) |
+| Disk            | 1TB NVMe                             |
+
+Best Known Configurations:
+
+```
+sudo bash run_clean.sh
+```
+
+## Benchmarking using automation scripts
+
+For your convinience, we prepare a set of automation scritps to help you download data, create docker, do data and model preprocessing, run accuracy, performance and compliance test in a batch. Please refer to ./automation/README.md for details about the usage. 
+Example for using automation scripts:
+```
+cd <THIS_REPO>/closed/Intel/code/automation/
+
```

Comment:
possible explain whole flow in details and use one model as an example? good to provide the commands that you ran for a model